python/lsst/pipe/tasks/compute_object_epochs.py

```diff
@@ -0,0 +1,133 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+from astropy.table import Table
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import numpy as np
+
+
+class ComputeObjectEpochsConnections(
+    pipeBase.PipelineTaskConnections,
+    dimensions=("tract", "skymap"),
+):
+    objectCat = pipeBase.connectionTypes.Input(
+        doc="Multiband catalog of positions in each patch.",
+        name="deepCoadd_obj",
+        storageClass="ArrowAstropy",
+        dimensions=["skymap", "tract", "patch"],
+        multiple=True,
+        deferLoad=True,
+        deferGraphConstraint=True,
+    )
+
+    epochMap = pipeBase.connectionTypes.Input(
+        doc="Healsparse map of mean epoch of objectCat in each band.",
+        name="deepCoadd_epoch_map_mean",
+        storageClass="HealSparseMap",
+        dimensions=("skymap", "tract", "band"),
+        multiple=True,
+        deferLoad=True,
+    )
+
+    objectEpochs = pipeBase.connectionTypes.Output(
+        doc="Catalog of epochs for objectCat objects.",
+        name="object_epoch",
+        storageClass="ArrowAstropy",
+        dimensions=["skymap", "tract", "patch"],
+        multiple=True,
+    )
+
+
+class ComputeObjectEpochsConfig(
+    pipeBase.PipelineTaskConfig,
+    pipelineConnections=ComputeObjectEpochsConnections,
+):
+    bands = pexConfig.ListField(
+        doc=("Bands in objectCat to be combined with `objectCat_selectors` to build objectCat column names."),
```

Comment:
I don't see `objectCat_selectors` anywhere in this file.

Reply:
Hmm. Perhaps Clare intended to use analysis_tools selectors and put it in the package for that reason, but that doesn't seem to have actually happened.